### ISSUE TITLE
Images not being sent properly to Chatwoot

### ISSUE THREAD

**Issue by WEBudoGT** [open-wa/wa-automate-nodejs#3114]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

EASY API/CLI

### Current Behavior

Images received from contacts are not shown on Chatwoot, and getting the following error on console:

`Something went wrong processing this message: false_XXXXXXXXXX@c.us_3EB069E953F5955FF12217 TypeError: Cannot read properties of undefined (reading 'id')
    at ChatwootClient.<anonymous> (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/integrations/chatwoot.js:497:97)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/integrations/chatwoot.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)`

### Steps To Reproduce

Send PNG or JPG image from any whatsapp account to open-wa

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.2318.11",
  "WA_AUTOMATE_VERSION": "4.65.0",
  "BROWSER_VERSION": "HeadlessChrome/111.0.5563.146",
  "START_TS": 1683129092893,
  "RAM_INFO": "Total: 16.79 GB | Free: 9.13 GB",
  "PPTR_VERSION": "19.11.1",
  "LATEST_VERSION": true,
  "CLI": true,
  "PATCH_HASH": "a6150",
  "LAUNCH_TIME_MS": 20179,
  "ACC_TYPE": "BUSINESS",
  "chats": 12,
  "messages": 18,
  "contacts": 1531,
  "isBiz": true,
  "isMd": true,
  "INSTANCE_ID": "d152dc17-d312-4d57-90fc-b2558380323e"
}
```

### Environment

```markdown
- OS: Linux 5.10
- Node: 19.8.1
- npm: 9.6.3
```

**Comment by smashah** [comment]
@WEBudoGT The code that is failing is this:

[contactReg[message.chatId] = (await this.createContact(message.sender)).id](https://github.com/open-wa/wa-automate-nodejs/blob/06e47bdd423780d69f8855ebda7bb3f4e67d2a03/src/cli/integrations/chatwoot.ts#L511)

The `createContact` promise is returning undefined. This is because the chatwoot request to create the contact failed. All chatwoot request failures would've been logged to the console.

Please try again with these CLI flags:

`--debug -v --disable-spins`

then check for the `CW REQ ERROR: ` message and share it here

**Comment by WEBudoGT** [comment]
I enabled debug but was unable to get any CW REQ ERROR on the logs.
All I got was this:

info: INCOMING MESSAGE ATTACHMENT 5XXXXXXXXX@c.us: undefined false_5XXXXXXXXX@c.us_7BFBEE6C5D748B660B3B1D7D2FB5E8BE {"timestamp":"2023-05-05T03:02:04.336Z"}
info: CW REQUEST: 200 get https://chatwoot-url/api/v1/accounts/1/conversations/33157/messages undefined {"timestamp":"2023-05-05T03:01:55.253Z"}
Something went wrong processing this message: false_5XXXXXXXXX@c.us_7BFBEE6C5D748B660B3B1D7D2FB5E8BE TypeError: Cannot read properties of undefined (reading 'name')
    at FormData._getContentDisposition (/usr/src/app/node_modules/form-data/lib/form_data.js:227:40)
    at FormData.append (/usr/src/app/node_modules/form-data/lib/form_data.js:71:21)
    at FormData._multiPartHeader (/usr/src/app/node_modules/form-data/lib/form_data.js:178:33)
    at ChatwootClient.<anonymous> (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/integrations/chatwoot.js:463:22)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/node_modules/@open-wa/wa-automate/dist/cli/integrations/chatwoot.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

I'm using docker, so maybe I used the wrong ENV vars?
WA_DEBUG: 'true'
WA_VERBOSE: 'true'

**Comment by smashah** [comment]
yes 

```
WA_DEBUG: 'true'
WA_VERBOSE: 'true'
```
will work also.

That seems to be a different error entirely.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.65.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.65.0...4.65.1) (2023-05-05)

- ⬆️ upgrade axios dep [`db23f05`](https://github.com/open-wa/wa-automate-nodejs/commit/db23f05fe18c1c3cf3f6532d4e337e68ababbeeb)
- Release 4.65.1 [`76a431a`](https://github.com/open-wa/wa-automate-nodejs/commit/76a431a2da81145ef64371bc51021b310a7247e0)
- 💚 fix ci build failure [`3ec5673`](https://github.com/open-wa/wa-automate-nodejs/commit/3ec5673c2832d36ba832b26f12decd8200407e51)
- 🐛 fix chatwoot file upload #3114 [`5c58e74`](https://github.com/open-wa/wa-automate-nodejs/commit/5c58e742350a4608634ce7d39c640d060e3de992)
- 🥚 patches update [`e3368e1`](https://github.com/open-wa/wa-automate-nodejs/commit/e3368e1bbfdb5c692c888a8ec80a01de05f78d76)
- 🥚 patches update [`07d57d1`](https://github.com/open-wa/wa-automate-nodejs/commit/07d57d1be2569da5f5f859087b97374c0ac8e7a6)
- 🥚 patches update [`c5ed644`](https://github.com/open-wa/wa-automate-nodejs/commit/c5ed64455ad17be11c242f2fcccb715ab48e6cfa)
- 🥚 patches update [`9760990`](https://github.com/open-wa/wa-automate-nodejs/commit/9760990552a302c387b29790da5e4b2d4acadd50)
- 🥚 patches update [`f871718`](https://github.com/open-wa/wa-automate-nodejs/commit/f8717183a82e41c9edae06e3df8e1403819f1d36)
- 🥚 patches update [`7b15c81`](https://github.com/open-wa/wa-automate-nodejs/commit/7b15c81ba28262beff023d54359ad4e8269a214e)
- 🥚 patches update [`d50104e`](https://github.com/open-wa/wa-automate-nodejs/commit/d50104ea98ac5498b04b636808351071aa9f8e55)
- 🥚 patches update [`35d36c9`](https://github.com/open-wa/wa-automate-nodejs/commit/35d36c98273c2d8430b8598895ff3252a8f7f6f5)
- 🥚 patches update [`b0ed76d`](https://github.com/open-wa/wa-automate-nodejs/commit/b0ed76d45c730604ba4d69632e55fce29870d2cd)
- updated types-only package [`fabe98f`](https://github.com/open-wa/wa-automate-nodejs/commit/fabe98fc49d4600e8c18072bc731a5446906434b)

**Comment by smashah** [comment]
fixed thanks
